subsys/bluetooth/mesh/Kconfig.rpl

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+choice BT_MESH_RPL_STORAGE_MODE
+	prompt "Replay protection list storage mode"
+
+config BT_MESH_RPL_STORAGE_MODE_EMDS
+	bool "Persistent storage of RPL in Emergency Data Storage"
+	depends on EMDS
+	help
+	  Persistent storage of RPL in Emergency Data Storage.
+
+endchoice
+
+if BT_MESH_RPL_STORAGE_MODE_EMDS
+
+config BT_MESH_RPL_INDEX
+	int "Unique index for the storage of RPL data"
+	default 999
```

Comment:
Is whole RPL has single index?

Reply:
Yes, all RPL data will be store as one single index/block.